lib/mwlib/src/MW/Media/Image/Imagick.php

```diff
@@ -110,12 +116,12 @@ public function scale( $width, $height, $fit = true )
 			if( $w <= $width && $h <= $height ) {
 				return $this;
 			}
-		}
 
-		try {
-			$this->image->resizeImage( $width, $height, \Imagick::FILTER_CUBIC, 0.8 );
-		} catch( \Exception $e ) {
-			throw new \Aimeos\MW\Media\Exception( $e->getMessage() );
+			try {
+				$this->image->resizeImage( $width, $height, \Imagick::FILTER_CUBIC, 0.8 );
+			} catch( \Exception $e ) {
```

Comment:
You should use only one try/catch block around everything

Reply:
Okay, will update the PR.